src/Fields/scans.jl

```diff
@@ -77,6 +77,7 @@ const ScannedComputedField = Field{<:Any, <:Any, <:Any, <:Scan}
 function compute!(field::ScannedComputedField, time=nothing)
     s = field.operand
     compute_at!(s.operand, time)
+    set_status!(field.status, time)
```

Comment:
Should you call this at the end, after the computation is performed? I guess if there is an error, the status will change but the field has not actually been computed